### PR TITLE
Bindingtestrootless

### PR DESCRIPTION
**Pull request by baude**
thanks to giuseppe!

### CODE REVIEW

File: pkg/rootless/rootless_linux.c [hunk 1]
```diff
@@ -200,6 +200,9 @@ can_use_shortcut ()
   if (argv == NULL)
     return false;
 
+  if (strstr (argv[0], "podman") == NULL)
```
Comment (rhatdan) [inline pkg/rootless/rootless_linux.c]: What happens if I execute /usr/bin/podman?  Should this me just checking for paths ending in /podman or podman?

Comment (jwhonce) [reply pkg/rootless/rootless_linux.c]: @rhatdan strstr() should only return NULL if "podman" is not contained in argv[0]

Comment (rhatdan) [reply pkg/rootless/rootless_linux.c]: No that is fine. My lying eyes saw strcmp now I see strstr...

Comment (giuseppe) [reply pkg/rootless/rootless_linux.c]: yes as long as podman is in the string then the process tries to join the user namespace.

I need to refactor this part and move the podman specific bits our of the rootless package

**Pull request merged**